Common/KeyMap.cpp

```diff
@@ -980,6 +985,7 @@ void AutoConfForPad(const std::string &name) {
 	// Add a couple of convenient keyboard mappings by default, too.
 	g_controllerMap[VIRTKEY_PAUSE].push_back(KeyDef(DEVICE_ID_KEYBOARD, NKCODE_ESCAPE));
 	g_controllerMap[VIRTKEY_UNTHROTTLE].push_back(KeyDef(DEVICE_ID_KEYBOARD, NKCODE_TAB));
+	g_controllerMap[VIRTKEY_FRAME_ADVANCE].push_back(KeyDef(DEVICE_ID_KEYBOARD, NKCODE_F));
```

Comment:
Hmm.  Right now it looks like once you start advancing, you're "trapped" into advancing.  In fact, if `frameStep_` isn't a member of EmuScreen, restarting the game won't even help.  That sounds dangerous, and doubly dangerous if we have a default key shortcut for the feature.

Maybe we should only have it enabled if you configure a key for it?

-[Unknown]